api/tests/Api/ContentNodes/DefaultLayout/UpdateDefaultLayoutTest.php

```diff
@@ -0,0 +1,49 @@
+<?php
+
+namespace App\Tests\Api\ContentNodes\DefaultLayout;
+
+use App\Tests\Api\ContentNodes\UpdateContentNodeTestCase;
+
+/**
+ * @internal
+ */
+class UpdateDefaultLayoutTest extends UpdateContentNodeTestCase {
+    public function setUp(): void {
+        parent::setUp();
+
+        $this->endpoint = '/content_node/default_layouts';
+        $this->defaultEntity = static::$fixtures['defaultLayout1'];
+    }
+
+    public function testPatchDefaultLayoutAcceptsValidJson() {
```

Comment:
do we need to update the DefaultLayout?
don't they all have the same items, and that should not change?

Reply:
Not at this time. Do you want me to remove it?